Return to Office/Building Access Appointments.md

```diff
@@ -0,0 +1,300 @@
+# Buildings Endpoints
+
+## General Setup
+
+All examples are using https://github.com/axios/axios#axios-api.
+```js
+axios = require('axios')
+PROOF_API_HOST = process.env.PROOF_API_HOST
+PROOF_API_TOKEN = process.env.PROOF_API_TOKEN
+```
+
+## POST: Creating a Building Access Appointment
+
+This endpoint creates a building access appointment (appointment) in the current user's government (based on the API token).
+
+**Currently not supported**
+<details>
+  <summary>will look something like this</summary>
+
+
+```js
+axios({
+  method: 'POST',
+  url: `${PROOF_API_HOST}/api/building_access_appointments`,
+  headers: {
+    'Accept': 'application/json',
+    'Authorization': `Bearer ${PROOF_API_TOKEN}`,
+    'Content-Type': 'application/json',
+  },
```

Comment:
I'm uncomfortable giving an example for an endpoint that is not supported

Reply:
It is hidden behind a spoiler tag ...

Reply:
![image](https://user-images.githubusercontent.com/23045206/100022842-1b8ad500-2da1-11eb-99bd-ceaa94bfb9e7.png)

Reply:
Yes, I understand... I stand by my stance to remove the snippet (albeit hidden by a spoiler tag), but... up to you.